src/data/faq_de.json

```diff
@@ -976,7 +976,9 @@
                                 "anchor": "check_in_qr_code_other_codes",
                                 "active": false,
                                 "textblock": [
-                                    "Ja, Sie können die Event-QR-Codes der luca-App mit der Corona-Warn-App scannen. Es ist derzeit nicht möglich die Event-QR-Codes von anderen Kontaktverfolgungs-Apps zu scannen. Genauere Informationen dazu im <a href='#check_in_luca' target='_self' >Abschnitt zur Mitnutzung der luca-QR-Codes</a>"
+                                    "<b>Aktualisiert am 15.07.2022</b>",
+                                    "Anmerkung: Die Eventregistrierung (Check-In-Funktionalität) der luca-App ist seit dem Frühjahr 2022 vorerst nicht mehr möglich. Dennoch können mit der Corona-Warn-App noch luca-QR-Codes gescannt werden.<hr>",

```

Comment:
I think all three variants are about equal for my ears. I'd rather say that the second option is the odd one out.

How about this:
```suggestion
                                    "Anmerkung: Die luca-App bietet seit dem Frühjahr 2022 keine Eventregistrierung (Check-In-Funktionalität) mehr an. Dennoch kann mit der Corona-Warn-App durch Scannen von luca-QR-Codes weiterhin eingecheckt werden.<hr>",

```

Reply:
@floscher 

Your suggestion is good:

Would it be clearer to say:
"Dennoch kann mit der Corona-Warn-App durch Scannen von bereits existierenden luca-QR-Codes weiterhin eingecheckt werden." ?

Reply:
@MikeMcC399 I think it's an unnecessary qualifier, since you can only ever scan QR codes that "already exist". If they don't exist, there's nothing to scan :wink:

Also luca seems to still produce compatible QR codes by default, so even future QR codes would be compatible. I haven't checked in a while, though if that's still the case.

Reply:
@floscher 
It's certainly unnecessary to change the text if luca can still produce compatible QR codes. I was unsure about this. I was really trying to refer to previously generated QR codes on the assumption that they can't be generated any more by luca, but if that is not the case, then please disregard what I wrote! 🙂